source/features/close-out-of-view-modals.tsx

```diff
@@ -5,7 +5,7 @@ import features from '../libs/features';
 const observer = new IntersectionObserver(([{intersectionRatio, target}]) => {
 	if (intersectionRatio === 0) {
 		observer.unobserve(target);
-		const dropdown = select(`
+		const dropdown = select<HTMLElement>(`
```

Comment:
Used the generic `HTMLElement` as I have no idea what kind of element this is supposed to be. I would guess a `<div />`, but not certain.